build-scripts/gulp/locale-data.js

```diff
@@ -24,8 +24,11 @@ const convertToJSON = async (
 ) => {
   let localeData;
   try {
+    // use "pt" for "pt-BR", because "pt-BR" is unsupported by @formatjs
+    const language = lang !== "pt-BR" ? lang : "pt";
```

Comment:
```suggestion
    const language = lang === "pt-BR" ? "pt" : lang;
```

Reply:
I did it like this, because in theory it is more efficient, when we know the the else case will be called just once.

Reply:
I don't think it change something performance wise. Also, this will be called at build time.

Reply:
yes, the impact will be minimal, but in theory it's faster :slightly_smiling_face: